### DISCUSSION TITLE
Does copilot take in dataview elements as context?

### DISCUSSION THREAD

**Discussion by stefanku** [logancyang/obsidian-copilot#209] [Q&A]
I have quite a few notes that are in fact built on dataview queries. Copilot does not seem to be able read this generated content. It probably just sees the javascript inside the md file. Is there a workaround? Or is it likely that content generated by dataview will at some point be supported as context?

**Comment by logancyang** [comment]
This is a possible enhancement, I'm adding this to the roadmap next.

**Reply by logancyang** [reply]
Also mentioned here https://github.com/logancyang/obsidian-copilot/issues/278#issuecomment-1925909940

**Comment by zanodor** [comment]
> I have quite a few notes that are in fact built on dataview queries. Copilot does not seem to be able read this generated content. It probably just sees the javascript inside the md file. Is there a workaround? Or is it likely that content generated by dataview will at some point be supported as context?

Have you tried markdownify-ing the query?
